### DISCUSSION TITLE
Cade connessione 2.4 Ghz con dispositivi domotici

### DISCUSSION THREAD

**Discussion by Fabx81** [indomus/forum#2276] [GENERALE - Discussioni generiche]
Salve,

dsa un paio di settimane ho il seguente problema: mi sveglio al mattino e vedo che è caduta la banda a 2.4 ghz della connessione. Funziona solo la 5ghz.

 

Pertanto, tutti i dispositivi domotici vanno offline.

 

Ho 3 echo dot: 2 di 4 generazione, uno di quinta.

 

Ho 8 lampadine Antela, 3 Extrastar e 2 prese Meross. Poi ho collegate anche TV LG, Lavatrice Samsung e 2 videocamere Blurams.

 

Il mio operatore è Fastweb e per loro la linea è ok. Infatti la connessione, di per se', va. Ma cade la 2.4ghz.

 

Ho fatto io quelli di Fastweb da remoto:

    riavvio del router (ho il Nexxt, quell con Alexa integrato, ma che non ho abilitato)
    factory reset del router
    disattivato il band steering
    riattivato il band steering
    ridotto a 6 ore il tempo impiegato per cercare i canali più liberi

Ho pensato a robe legate agli indirizzi IP dei dispositivi, al DHCP, alla subnet mask, ma non so proprio come orientarmi e i tecnici di Fastweb sono preparati su quelle 3-4 cose che ero riuscito a fare già in autonomia.

 

Grazie :-)

**Comment by snakuzzo** [comment]
Se casca proprio l'access point 2.4 GHz del router, tutto quello che hai menzionato non c'entra niente -> Cambia router.

**Comment by Fabx81** [comment]
Anzitutto grazie per la risposta. Dopo di che: perchè casca?

**Reply by snakuzzo** [reply]
e io che ne so ? non ho la sfera di cristallo, né strumenti per poter vedere perché casca

**Reply by snakuzzo** [reply]
controlla i log del router, se puoi... e vedi se c'è qualcosa che può darti indicazioni....altrimenti ciccia

**Reply by Fabx81** [reply]
Come faccio a controllare i log del router dal Nexxt? 

EDIT:  ho trovato nel tab Diagnostica, la voce "Messaggi debug". Crash log, Scarica registro. 
Mi scarica un file .rar. Che contiene una cartella tmp. Questa cartella è vuota.

**Comment by KC-inDomus** [comment]
> Salve,
> 
> 
> 
> dsa un paio di settimane ho il seguente problema: mi sveglio al mattino e vedo che è caduta la banda a 2.4 ghz della connessione. Funziona solo la 5ghz.
> 
> 
> 
>  
> 
> 
> 
> Pertanto, tutti i dispositivi domotici vanno offline.
> 
> 
> 
>  
> 
> 
> 
> Ho 3 echo dot: 2 di 4 generazione, uno di quinta.
> 
> 
> 
>  
> 
> 
> 
> Ho 8 lampadine Antela, 3 Extrastar e 2 prese Meross. Poi ho collegate anche TV LG, Lavatrice Samsung e 2 videocamere Blurams.
> 
> 
> 
>  
> 
> 
> 
> Il mio operatore è Fastweb e per loro la linea è ok. Infatti la connessione, di per se', va. Ma cade la 2.4ghz.
> 
> 
> 
>  
> 
> 
> 
> Ho fatto io quelli di Fastweb da remoto:
> 
> 
> 
>     riavvio del router (ho il Nexxt, quell con Alexa integrato, ma che non ho abilitato)
> 
>     factory reset del router
> 
>     disattivato il band steering
> 
>     riattivato il band steering
> 
>     ridotto a 6 ore il tempo impiegato per cercare i canali più liberi
> 
> 
> 
> Ho pensato a robe legate agli indirizzi IP dei dispositivi, al DHCP, alla subnet mask, ma non so proprio come orientarmi e i tecnici di Fastweb sono preparati su quelle 3-4 cose che ero riuscito a fare già in autonomia.
> 
> 
> 
>  
> 
> 
> 
> Grazie :-)

se ti "cade la banda" nel senso che smette di trasmettere l'antenna, allora è un problema hardware.
diversamente, potrebbe essere [un problema di interferenze](https://indomus.it/focus/canali-wi-fi-scegliere-al-meglio-per-le-massime-prestazioni/).

**Reply by Fabx81** [reply]
> > Salve,
> > dsa un paio di settimane ho il seguente problema: mi sveglio al mattino e vedo che è caduta la banda a 2.4 ghz della connessione. Funziona solo la 5ghz.
> >  
> > Pertanto, tutti i dispositivi domotici vanno offline.
> >  
> > Ho 3 echo dot: 2 di 4 generazione, uno di quinta.
> >  
> > Ho 8 lampadine Antela, 3 Extrastar e 2 prese Meross. Poi ho collegate anche TV LG, Lavatrice Samsung e 2 videocamere Blurams.
> >  
> > Il mio operatore è Fastweb e per loro la linea è ok. Infatti la connessione, di per se', va. Ma cade la 2.4ghz.
> >  
> > Ho fatto io quelli di Fastweb da remoto:
> > ```
> > riavvio del router (ho il Nexxt, quell con Alexa integrato, ma che non ho abilitato)
> > 
> > factory reset del router
> > 
> > disattivato il band steering
> > 
> > riattivato il band steering
> > 
> > ridotto a 6 ore il tempo impiegato per cercare i canali più liberi
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Ho pensato a robe legate agli indirizzi IP dei dispositivi, al DHCP, alla subnet mask, ma non so proprio come orientarmi e i tecnici di Fastweb sono preparati su quelle 3-4 cose che ero riuscito a fare già in autonomia.
> >  
> > Grazie :-)
> 
> se ti "cade la banda" nel senso che smette di trasmettere l'antenna, allora è un problema hardware. diversamente, potrebbe essere [un problema di interferenze](https://indomus.it/focus/canali-wi-fi-scegliere-al-meglio-per-le-massime-prestazioni/).

come faccio a capire se smette di funzionare l'antenna o altro?

**Reply by snakuzzo** [reply]
Controlla i log (se puoi) e vedi se trovi qualcosa di indicativo, di più non so dirti

**Reply by Fabx81** [reply]
> Controlla i log (se puoi) e vedi se trovi qualcosa di indicativo, di più non so dirti

Come faccio a controllare i log del router dal Nexxt?

EDIT: ho trovato nel tab Diagnostica, la voce "Messaggi debug". Crash log, Scarica registro.
Mi scarica un file .rar. Che contiene una cartella tmp. Questa cartella è vuota.

**Reply by snakuzzo** [reply]
Non so che dirti

**Reply by Fabx81** [reply]
> > Salve,
> > dsa un paio di settimane ho il seguente problema: mi sveglio al mattino e vedo che è caduta la banda a 2.4 ghz della connessione. Funziona solo la 5ghz.
> >  
> > Pertanto, tutti i dispositivi domotici vanno offline.
> >  
> > Ho 3 echo dot: 2 di 4 generazione, uno di quinta.
> >  
> > Ho 8 lampadine Antela, 3 Extrastar e 2 prese Meross. Poi ho collegate anche TV LG, Lavatrice Samsung e 2 videocamere Blurams.
> >  
> > Il mio operatore è Fastweb e per loro la linea è ok. Infatti la connessione, di per se', va. Ma cade la 2.4ghz.
> >  
> > Ho fatto io quelli di Fastweb da remoto:
> > ```
> > riavvio del router (ho il Nexxt, quell con Alexa integrato, ma che non ho abilitato)
> > 
> > factory reset del router
> > 
> > disattivato il band steering
> > 
> > riattivato il band steering
> > 
> > ridotto a 6 ore il tempo impiegato per cercare i canali più liberi
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Ho pensato a robe legate agli indirizzi IP dei dispositivi, al DHCP, alla subnet mask, ma non so proprio come orientarmi e i tecnici di Fastweb sono preparati su quelle 3-4 cose che ero riuscito a fare già in autonomia.
> >  
> > Grazie :-)
> 
> se ti "cade la banda" nel senso che smette di trasmettere l'antenna, allora è un problema hardware. diversamente, potrebbe essere [un problema di interferenze](https://indomus.it/focus/canali-wi-fi-scegliere-al-meglio-per-le-massime-prestazioni/).

Ciao, grazie per la risposta. 

Allora, ho letto il link che mi hai scritto. 

All'inizio leggo "La Wi-Fi domestica, sistema nervoso della nostra [domotica personale](https://indomus.it/focus/definizioni-domotica-personale/) e non solo, lavora sostanzialmente su due gamme di frequenze radio, **la 2.4 GHz (802.11b, g, n)** e la 5 GHz (802.11n, ac e ax)" 

Nel pannello del mio modem Nexxt vedo che anche la 2.4ghz ha come modalità wifi 802.11g/n/ax invece che 802.11b, g, n. Può essere un problema questo? 

Per quanto concerne i canali, io vedo che di default il mio modem ha impostato il canale 11 per la 2.4 ghz. Ha senso provare 1 e 6? Ad ogni modo, ho scaricato Fing, come indicato nel link e vedo un po'..... 

Altre cose: ho visto nella schermata di diagnosi del modem che la 2.4ghz risulta "On", ma io non la vedo sui miei dispositivi. Sai dirmi come mai?

**Reply by snakuzzo** [reply]
Ricordo che su un vecchio router avevo problemi solo con un device utilizzando `802.11 b/g/n`,  Impostato `802.11 b/g` ho risolto.
Ad oggi è di nuovo su `802.11 b/g/n` e non ho alcun problema.
Più che dirti "prova a smanettare con quei parametri" non so dirti.
Ripeto quanto ti ho già detto in precedenza, io non posso sapere del perché tu non vedi la tua WIFI o cose simili...inutile continuare a fare domande in quel senso.

**Reply by Fabx81** [reply]
> > Salve,
> > dsa un paio di settimane ho il seguente problema: mi sveglio al mattino e vedo che è caduta la banda a 2.4 ghz della connessione. Funziona solo la 5ghz.
> >  
> > Pertanto, tutti i dispositivi domotici vanno offline.
> >  
> > Ho 3 echo dot: 2 di 4 generazione, uno di quinta.
> >  
> > Ho 8 lampadine Antela, 3 Extrastar e 2 prese Meross. Poi ho collegate anche TV LG, Lavatrice Samsung e 2 videocamere Blurams.
> >  
> > Il mio operatore è Fastweb e per loro la linea è ok. Infatti la connessione, di per se', va. Ma cade la 2.4ghz.
> >  
> > Ho fatto io quelli di Fastweb da remoto:
> > ```
> > riavvio del router (ho il Nexxt, quell con Alexa integrato, ma che non ho abilitato)
> > 
> > factory reset del router
> > 
> > disattivato il band steering
> > 
> > riattivato il band steering
> > 
> > ridotto a 6 ore il tempo impiegato per cercare i canali più liberi
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Ho pensato a robe legate agli indirizzi IP dei dispositivi, al DHCP, alla subnet mask, ma non so proprio come orientarmi e i tecnici di Fastweb sono preparati su quelle 3-4 cose che ero riuscito a fare già in autonomia.
> >  
> > Grazie :-)
> 
> se ti "cade la banda" nel senso che smette di trasmettere l'antenna, allora è un problema hardware. diversamente, potrebbe essere [un problema di interferenze](https://indomus.it/focus/canali-wi-fi-scegliere-al-meglio-per-le-massime-prestazioni/).

Grazie. 
Ho aspettato a rispondere perchè avevo cambiato il canale (messo l'11, per la 2.4ghz) e avevo anche messo la frequenza a 20 invece che i 20/40 default. Per cirfca 5 giorni è andato tutto liscio. Poi stamane verso le 11.30 c'è stata una nuova caduta della 2.4 ghz. Ho riavviato il router e tutto ok. 

Non è una roba di antenna (credo) perchè la 5ghz tiene.

**Reply by snakuzzo** [reply]
Io ho avuto qualche mese fa un problema simile con il Fritzbox.
L'ho "buttato" e ho risolto :)

**Reply by Fabx81** [reply]
> Io ho avuto qualche mese fa un problema simile con il Fritzbox. L'ho "buttato" e ho risolto :)

Il fritzbox suppo0ngo sia un modem/router, giusto?

**Reply by snakuzzo** [reply]
Sì

**Reply by KC-inDomus** [reply]
sì - https://indomus.it/componenti/recensione-avm-fritzbox-7530-ax/

**Comment by Fabx81** [comment]
Grazie. 
Ad ogni modo aggiorno: stanotte ho avuto ancora il problema. Alchè ho cambiato modem. 
Ora ho su il Fastgate (il predecessore Fastweb del Nexxt che ho usato da quando ho il problema). E quindi vediamo come va. 

La cosa curiosa è che prima avevo messo su un mio D-Link DSL 3682. Ebbene, con quel modem, vedevo 2.6 e 5ghz (erano le relative luci sull'apparecchio), ma non c'era internet.

**Reply by snakuzzo** [reply]
E questa cosa che c'entra con tutto il resto?
Non confondere gli argomenti

**Comment by Fabx81** [comment]
C'entra che, tra i consigli datimi, c'è quello di cambiare modem. 

Uno di questi ha dato il problema che ho detto e l'ho riportato. Ho dato un elemento. Se è irrilevante, amen.

**Reply by snakuzzo** [reply]
mi riferivo alle "luci sul router"
P.S. per cortesia fai reply sotto l'ultimo messaggio, non creare nuovi thread come hai fatto ora. Grazie

**Reply by Fabx81** [reply]
Era per comprovare il funzionamento delle due bande.

**Comment by Fabx81** [comment]
Alla fine era il modem.
